vcpkg/concepts/features.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Understanding features in vcpkg
+description: This article explains the concept of features, their capabilities, and their expected contents in detail.
+author: JavierMatosD
+ms.author: javiermat
+ms.date: 02/21/2024
+ms.topic: concept-article
+---
+
+# Concept: features
+
+## Features
+
+Features represent sets of functionality, behavior, and dependencies that can be selectively added to a package or project upon installation. 
+
+By design, features should follow these principles:
+
+- Additive: enabling feature A should provide new functionality otherwise missing from the package without disabling any other functionality.
+- Non-exclusive: enabling feature A should not prevent other features from being installed.
+
+As such, features should not be used to define alternative sets of functionalities. For example, a graphics library should not use features to choose between exclusive graphics backends, since it is not possible to install all of them at the same time.
```

Comment:
```suggestion
Features should not be used to define alternative sets of functionalities. For example, a graphics library should not use features to choose between exclusive graphics backends, since it's not possible to install all of them at the same time.
```